app/pb_kits/playbook/pb_title_detail/_title_detail.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= content_tag(:div,
     id: object.id,
     data: object.data,
-    class: object.classname(object.kit_class)) do %>
-  <%= object.title %>
-  <%= object.detail %>
+    class: object.classname) do %>
+  <%= pb_rails("title", props: { text: object.title, size: 4 }) if object.title %>
+  <%= pb_rails("body", props: { text: object.detail, color: "light" }) if object.detail %>
```

Comment:
It looks like the previous use of the `body` kit was passing the `detail` as a block. Does this work the same way for this use case?

Reply:
No - this one passes the title and detail text to kits that will render if passed a string (defaulting to nil, and not rendering without one)